pkg/solana/chain.go

```diff
@@ -12,6 +12,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/gagliardetto/solana-go"
 	solanago "github.com/gagliardetto/solana-go"
```

Comment:
I think the import already existed under the `solanago` alias. Think this was aliased to avoid naming collision with the package